### PR TITLE
fix(frontend): dream mall button: Style Edit Table

### PR DESCRIPTION
**Pull request by sebastian2357**
### Issues

- DM Button: Style Edit Table #1919

### Todo

- [X] Move colors to themes
- [X] new requirement: "add users" should only be visible for table-owner

### CODE REVIEW

**Review by Bettelstab — APPROVED** [review]
Please have a look at the proposed simplifications. We should not do our own route parameter parsing. But everything works, looks good and hence I approve!

File: frontend/src/stores/tablesStore.ts [hunk 1]
```diff
@@ -135,8 +134,10 @@ export const useTablesStore = defineStore(
     }
 
     const existsMyTable = computed(() => myTable.value !== null)
-
     const defaultMyTableName = computed(() => currentUser.value?.name ?? '')
+    const isTableChangeable = (id: number): boolean => {
+      return existsMyTable.value && myTable.value?.id === id
```
[the remark this answers is not in the document]
Comment (sebastian2357) [reply frontend/src/stores/tablesStore.ts:139]: fixed

File: frontend/src/components/malltalk/settings/TableSettingsRoot.vue [hunk 2]
```diff
@@ -1,38 +1,83 @@
...
+      ]
+    : []),
+])
+
+const leaveTable = () => {
+  navigate('/')
+}
 </script>
+
+<style scoped lang="scss">
```
[the remark this answers is not in the document]
Comment (sebastian2357) [reply frontend/src/components/malltalk/settings/TableSettingsRoot.vue:74]: if I remove this then the variables are not valid any more.

File: frontend/src/components/malltalk/settings/TableSettingsRoot.vue [hunk 3]
```diff
@@ -1,38 +1,83 @@
...
+
 defineProps<StepProps>()
 const emit = defineEmits<StepEmits>()
 
 const onClick = (stepId: string) => {
   emit('goTo', stepId)
 }
+
+const pageContext = usePageContext()
+const tableId = computed(() => {
```
[the remark this answers is not in the document]
Comment (sebastian2357) [reply frontend/src/components/malltalk/settings/TableSettingsRoot.vue:41]: Ah, that is much better - Thank you!

**Pull request merged**